### ISSUE TITLE
v3 Type error: 'parseISO' cannot be used as a value because it was exported using 'export type'.

### ISSUE THREAD

**Issue by nemchik** [date-fns/date-fns#3615]
In v2 I had
```ts
import { format, parseISO } from "date-fns";
```

In v3 I have tried the above and also
```ts
import { format } from "date-fns/format";
import { parseISO } from "date-fns/parseISO";
```

Both give me this in v3
```
Type error: 'parseISO' cannot be used as a value because it was exported using 'export type'.
Type error: 'format' cannot be used as a value because it was exported using 'export type'.
```

**Comment by naphtul** [comment]
Also true for `parse`.
Would love to get the new usage.

**Comment by b-maslennikov** [comment]
Issue #3588

**Comment by nstuyvesant** [comment]
Try going to node_modules/date-fns/package.json and doing a global replace of "*.mts" with "*.ts" and the problem with go away.

There was never a need to create the .d.mts files for ESM as the .d.ts files can be used without issues.

**Comment by kossnocorp** [comment]
Can someone please produce a minimal reproduction repo? It will help to fix the problem.

We have a set of different environments [where it's all good](https://github.com/date-fns/date-fns/actions/runs/7247215749/job/19740705115): https://github.com/date-fns/date-fns/tree/main/examples

Sorry that it slipped!

**Comment by kossnocorp** [comment]
I shipped with fix with `date-fns@3.0.1`, please try it
